libindy/include/indy_ledger.h

```diff
@@ -1256,6 +1256,57 @@ extern "C" {
                                                                                       const char*   request_json)
                                                                 );
 
+    /// Builds a LEDGERS_FREEZE request. Request to freeze list of ledgers.
+    ///
+    /// #Params
+    /// command_handle: command handle to map callback to caller context.
+    /// submitter_did: (Optional) DID of the read request sender (if not provided then default Libindy DID will be used).
+    /// ledgers_ids: list of ledgers IDs for freezing ledgers (json format).
+    /// cb: Callback that takes command result as parameter.
+    ///
+    /// #Returns
+    /// Request result as json.
+    ///
+    /// #Errors
+    /// Common*
+    extern indy_error_t indy_build_ledgers_freeze_request(indy_handle_t command_handle,
+                                                                 const char *  submitter_did,
+                                                                 const char *  ledgers_ids,
```

Comment:
Why do we use json format for ledgers_ids here?

Reply:
It need for API libindy.